deploy/bootstrap-docker.sh

```diff
@@ -13,7 +13,7 @@ sudo chgrp circleci /creds/sa-key.json
 sudo chmod g+r /creds/sa-key.json
 sudo chown -R circleci ~/.gradle
 sudo mkdir -p ~/.cache/yarn
```

Comment:
why remove 
`sudo chown -R circleci ~/.cache/yarn`

Reply:
-R == recursive and I'm now `chown`ing its parent